From 86dec4828fdba9602bce7795ffe4b733f0677eaf Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 24 Jun 2014 15:03:12 +0000
Subject: [PATCH 0475/1000] MANTIS 0019643 extension:
 Platform_Extension_CUSS_1_2: Single-application mode device support
 platform.idl V1.9

code resetMediaRecord(in mRealComponents mediaOffRoller);
same as resetDevices; except for mediaOffRoller components
the mediaOffRoller components will playback the stored events



git-svn-id: svn://localhost/SelfServiceCommon/trunk@689 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 0b86260c..f57950f8 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "1.8";
+    const string interface_version = "1.9";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -68,6 +68,7 @@ module PlatformManagement
     };
 
     typedef sequence<mEnvironmentComponent> mEnvironmentComponents;
+    typedef sequence<string> mRealComponents;
 
     enum PowerStatus
     {
@@ -140,9 +141,10 @@ module PlatformManagement
         */
         code beginMediaRecord(in string contextName);
         
-        /** delete events, recorded for media-roll-off mode
+        /** same as resetDevices; except for mediaOffRoller components
+            the mediaOffRoller components will playback the stored events
         */
-        code resetMediaRecord();
+        code resetMediaRecord(in mRealComponents mediaOffRoller);
     };
 
     enum DataStatusExtension
-- 
2.41.0.windows.1

